packages/documentation/src/stories/components/icons/getting-started.docs.mdx

```diff
@@ -107,4 +107,4 @@ All icons are also available from a CDN, which is the default if you don't confi
 ### Check the Content Security Policy
 
 If your project includes the <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy" target="_blank">Content-Security-Policy</a> response header (usually in `index.html`),
-make sure that `default-src` is set to `'self'`.
+make sure that `default-src` is set to `'self'`. When using the Post-Icon component, icons are retrieved from [`https://unpkg.com/`](https://unpkg.com/) therefore this URL should be added to the `connect-src` in index.html.
```

Comment:
"Make sure it is set to `self`" and "therefore this URL should be added to the `connect-src` in index.html." are two contradictory statements.
Please add one clear and easy to understand statement or even better, create an example code snippet, which shows [how to set the meta-tag in the header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#using_the_html_meta_element).